contrib/IECoreUSD/src/IECoreUSD/SceneCacheData.h

```diff
@@ -0,0 +1,151 @@
+#ifndef IECOREUSD_SCENE_CACHE_DATA_H
```

Comment:
Missing license - looks like there are more of these so I'll stop commenting them.

Reply:
addressed in place